common.gradle

```diff
@@ -1,6 +1,6 @@
 // Common definitions and constraints to be applied to the projects.
 // These are applied in the buildscript block.
-def useMavenLocal = false
+def useMavenLocal = true
```

Comment:
Note to self: switch this back when https://github.com/DataBiosphere/terra-cloud-resource-lib/pull/92 is merged and published.